receiver/cloudfoundryreceiver/converter.go

```diff
@@ -74,3 +116,29 @@ func copyEnvelopeAttributes(attributes pcommon.Map, envelope *loggregator_v2.Env
 		attributes.PutStr(attributeNamePrefix+"instance_id", envelope.InstanceId)
 	}
 }
+
+func getEnvelopeDatapointAttributes(envelope *loggregator_v2.Envelope) pcommon.Map {
```

Comment:
This comment is a bit particular, but given the function is being applied to all telemetry types (even traces in the future), maybe is better thinking of a new name (??), maybe something like `getEvenvelopeDataAttributes` ?